examples/faster_rcnn/README.md

```diff
@@ -2,11 +2,25 @@
 
 ### Performance
 
-| Training Setting | Evaluation | Reference Implementation | ChainerCV |
+| Training Setting | Evaluation | Reference | ChainerCV |
 |:-:|:-:|:-:|:-:|
 | VOC 2007 trainval | VOC 2007 test|  69.9 mAP [1] | 70.5 mAP |
 
 
+### Speed
+
+Speed of the Caffe implementation and ChainerCV implementation are almost same in the test mode.
+We compared the time it takes to forward an image.
+
+We compared `chainercv.links.FasterRCNN.__call__` and [Caffe's equivalent function](https://github.com/rbgirshick/py-faster-rcnn/blob/master/lib/fast_rcnn/test.py#L154).
+We used [an image](https://github.com/rbgirshick/py-faster-rcnn/blob/master/data/demo/000456.jpg) from PASCAL VOC 2007 test split.
+Our experiment was conducted on Ubuntu 14.04.5 with Pascal Titan X.
+
+| py-faster-rcnn | ChainerCV |
```

Comment:
Yes, they both refer to the same implementation.
I did not want to say `py-faster-rcnn` at Performance section because it can contain results from different implementations.
For example, ResNet implementation is not included in py-faster-rcnn.

Anyway, I think this can be anything until we have different results to post.